### ISSUE TITLE
Docs: OlsDatasetExtractor: CSV not generated

### ISSUE THREAD

**Issue by joeflack4** [EBISPOT/OXO#62]
## Description
I'm trying to learn about OXO and I have set up a local instance using docker. I'm following the docs, and I want to create `datasets.csv` and follow the rest of the documentation walkthrough. But either (a) the file is not being generated, or (b) I can't find it.

This is the command that I'm running:
```sh
docker run --net=host -v $(pwd)/config.ini:/mnt/config.ini -v $(pwd)/idorg.xml:/mnt/idorg.xml \
    -v oxo-neo4j-import:/mnt/neo4j -it ebispot/oxo-loader:stable \
        python /opt/oxo-loader/OlsDatasetExtractor.py \
            -c /mnt/config.ini -i /mnt/idorg.xml -d /mnt/neo4j/datasources.csv
```

## Expected behavior

**Comment by henrietteharmse** [comment]
@joeflack4 Thank you for logging the issue in such detail. We will look into it as soon as we can.

**Comment by matentzn** [comment]
We should probably unify custom deployments in https://github.com/EBISPOT/ontotools-docker

These kinds of issues are mostly problems with mount points etc.. @joeflack4 did you try using the docker compose setup?

**Comment by henrietteharmse** [comment]
@matentzn [https://github.com/EBISPOT/ontotools-docker](https://github.com/EBISPOT/ontotools-docker) addresses the complete Ontology Tools stack. However, not everyone wants to install the complete stack.

**Comment by matentzn** [comment]
I agree, I was more thinking of moving the docs there, and using docker compose even for the individual services; it would just be less support work in the future, bypassing typical mistakes in Following readme docs.

**Comment by joeflack4** [comment]
Hey guys. Thanks for looking into this!

@matentzn Yep, I was just following the documentation. It asks to do `docker-compose up`, so yep, I called that from the root directory.

**Comment by udp** [comment]
It seems the paths are wrong in the supplied config.ini.

```
[Paths]
exportFileDatasources=datasources.csv
exportFileTerms=/path/terms.csv
exportFileMappings=/path/mappings.csv
```

should be


```
[Paths]
exportFileDatasources=/mnt/neo4j/datasources.csv
exportFileTerms=/mnt/neo4j/terms.csv
exportFileMappings=/mnt/neo4j/mappings.csv
```

I will get this updated. As @matentzn says we should probably in the long term unify all of these separate Docker instructions into the ontotools-docker repository, which is more up to date but currently only deploys the whole stack.

**Comment by joeflack4** [comment]
Makes sense that that is the issue! For now, Nico has instructed me to use the full ontotools stack for my purposes, as it should serve my purposes just fine. But thanks for looking into this.
